index.html

```diff
@@ -14,8 +14,16 @@
 
 <body>
     <header>
-        <nav></nav>
+        <img src="MacintoshHD/Users/jonathanreeves/lamdbaschools/Sprint-Challenge--UI-Responsive/img/lambda-black.png">
```

Comment:
You were very close here. Try to specify just the relative path of the css file from your html file > ``` ../img/lambda-black.png ```

Reply:
That helped. Thank you. I couldn't figure out why it wasn't working.